certified-connectors/Tribal - Maytas/apiProperties.json

```diff
@@ -0,0 +1,120 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "[DUMMY]",
+          "scopes": [
+            "Maytas.UI",
+            "MaytasGateway.Connector.MetaData",
+            "web_hooks",
+            "edge",
+            "openid",
+            "edge_identity",
+            "offline_access"
+          ],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrlTemplate": {
+              "value": "https://identity{environment}/{region}/ids/{tenantId}/connect/authorize"
+            },
+            "tokenUrlTemplate": {
+              "value": "https://identity{environment}/{region}/ids/{tenantId}/connect/token"
+            },
+            "refreshUrlTemplate": {
+              "value": "https://identity{environment}/{region}/ids/{tenantId}/connect/token"
+            }
+          }
+        }
+      },
+      "token:environment": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 0,
+            "required": "true",
+            "allowedValues": [
+              {
+                "text": "Live",
+                "value": ".tribaledge.com"
+              },
+              {
+                "text": "Development",
+                "value": "-master.edge.tribaldev.net"
+              }
+            ]
+          },
+          "description": "Environment",
+          "displayName": "Environment",
+          "tooltip": "Environment"
+        }
+      },
+      "token:region": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 0,
+            "required": "true",
+            "allowedValues": [
+              {
+                "text": "Emea",
+                "value": "emea"
+              },
+              {
+                "text": "Apac",
+                "value": "apac"
+              }
+            ]
+          },
+          "description": "Region",
+          "displayName": "Region",
+          "tooltip": "Region"
+        }
+      },
+      "token:tenantId": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 2,
+            "required": "true"
```

Comment:
Please add allowedValues for token:tenantId, this must resolve the api properties validation error.

Reply:
I want the tenant id to be a free text field, wouldn't adding allowedValues restrict it just to a select number of options?

Reply:
It is allowed, I will approve.